src/main/app.ts

```diff
@@ -76,6 +77,10 @@ if (toBoolean(config.get<boolean>('featureToggles.testingSupport'))) {
   new TestingSupportFeature().enableFor(app)
 }
 
+moment.prototype.toISOString = function () {
```

Comment:
@dharmendrak can you please add comment explaining why we doing this? Someone can be very surprised we overriding default implementation.